content/javascript/concepts/storage/terms/setItem.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: '.setItem()' 
```

Comment:
looks like this isn't in the correct location. It should be within a folder named "setItem"